pkg/testhelper/testhelper.go

```diff
@@ -106,6 +106,7 @@ const (
 	ReasonForCompliance             = "Reason For Compliance"
 	Category                        = "Category"
 	RoleBindingName                 = "Role Binding Name"
+	ClusterRoleBindingName          = "Cluster Role Binding Name"
```

Comment:
Should this be `Cluster Role Name`?  I think the roleRef references the Cluster Role itself.

![image](https://github.com/test-network-function/cnf-certification-test/assets/4563082/8dbab72f-8f7b-4f55-b752-78269a1e200e)

Reply:
https://kubernetes.io/docs/reference/access-authn-authz/rbac/#clusterrolebinding-example

Reply:
In this case, the name of the cluster role the binding is referencing would be 'secret-reader'.